ThunderCloud/BundleDiagnosticsTableViewController.swift

```diff
@@ -0,0 +1,175 @@
+            inBackgroundRow.value = ContentController.showFeedbackInBackground
+            inBackgroundRow.valueChangeHandler = { (value, _) in
+                guard let showFeedback = value as? Bool else { return }
+                ContentController.showFeedbackInBackground = showFeedback
+            }
+            
+            settingsRows.append(inBackgroundRow)
+        }
+        
+        let downloadOnWifiRow = StormDiagnosticsSwitchRow(title: "Only Download on WiFi", subtitle: "The app will only download delta updates if your phone is connected to a WiFi network", id: "wifi")
+        downloadOnWifiRow.value = ContentController.onlyDownloadOverWifi
+        downloadOnWifiRow.valueChangeHandler = { (value, _) in
+            guard let onlyDownloadOverWifi = value as? Bool else { return }
+            ContentController.onlyDownloadOverWifi = onlyDownloadOverWifi
+        }
+        settingsRows.append(downloadOnWifiRow)
+        
+        data = [
+            TableSection(rows: [buildDateRow], header: "Build Information"),
+            TableSection(rows: [bundleTimestampRow, deltaTimestampRow], header: "Timestamps"),
+            TableSection(rows: settingsRows, header: "Settings"),
+            TableSection(rows: [deleteDeltaRow, switchRow], header: "Actions")
+        ]
+    }
+    
+    private func deleteDeltaBundle() {
+        
+        let confirmationAlert = UIAlertController(
+            title: "Delete Delta Bundle",
+            message: "This will delete the delta bundle and cannot be undone. The app will be killed after this is completed. Are you sure you wish to proceed?\n\nPlease note this will not stop the delta being re-downloaded on next launch or in the background.",
+            preferredStyle: .alert
+        )
+        confirmationAlert.addAction(UIAlertAction(title: "Yes, delete", style: .destructive, handler: { (action) in
+            
+            guard let deltaDirectory = ContentController.shared.deltaDirectory else {
+                return
+            }
+            
+            ContentController.shared.removeBundle(in: deltaDirectory)
+            
+            let content = UNMutableNotificationContent()
+            content.title = "Delta Deleted"
+            content.body = "Tap here to re-launch the app"
+            
+            let trigger = UNTimeIntervalNotificationTrigger(timeInterval: 3, repeats: false)
+            
+            // Create the request
+            let uuidString = UUID().uuidString
+            let request = UNNotificationRequest(identifier: uuidString, content: content, trigger: trigger)
+            
+            // Schedule the request with the system.
+            let notificationCenter = UNUserNotificationCenter.current()
+            notificationCenter.add(request, withCompletionHandler: nil)
+            
+            DispatchQueue.main.asyncAfter(deadline: .now() + 2) {
+                exit(-1)
```

Comment:
Might need to brush up on my C, and not sure this really matters too much :'). But do we want to exit with success as this was the intended behavior. I.e. with a `0`